React-Core.podspec

```diff
@@ -47,7 +47,6 @@ Pod::Spec.new do |s|
   s.compiler_flags         = folly_compiler_flags + ' ' + boost_compiler_flags
   s.header_dir             = "React"
   s.framework              = "JavaScriptCore"
-  s.library                = "stdc++"
```

Comment:
nice! we should create a todo bug to clean up our reactnative comunity dependnecy in office codebase right?

Reply:
Not sure why we would if we want to use react-native-community modules. But if there's a compelling reason, we can talk about this elsewhere.